CONTRIBUTING.rst

```diff
@@ -176,6 +176,20 @@ Imports
 
   Is the *only* correct sequence of import statements for these symbols.
 
+* We carefully selected the ordering criteria to match those implemented by
+  PyCharm. PyCharm's "Optimize Imports" feature should be the preferred method
+  of resolving import statement ordering violations, as the line numbers
+  reported by our flake8 plugin are not always optimal in illuminating the
+  nature of the violations.
+
+* The one violation *not* addressable via PyCharm is our requirement that
+  single-symbol ``from`` imports be wrapped the same as multi-symbol ones.
+  Currently, this must be corrected manually. Vim users may find the following
+  macro convenient for this purpose:
+  ::
+
+  ^3wi(<ENTER><ESCAPE>A,<ENTER>)
```

Comment:
The other pre blocks are indented. I think this one shoul dbe too for consistency